api/docker/php/docker-entrypoint.sh

```diff
@@ -55,13 +55,25 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 			echo "The database is now ready and reachable"
 		fi
 
+		if [ "$APP_ENV" != 'prod' ]; then
+			echo "Create test database"
+			bin/console doctrine:database:create --if-not-exists -e test
```

Comment:
Shouldn't it be the purpose of a trait in Behat/PHPUnit?

Reply:
Even creating the database and loading the fixtures?
Have you already done that somewhere?

Reply:
The test database could (should?) be created on demand, the schema and its fixtures could be initialized and loaded thanks to the [RefreshDatabaseTrait](https://github.com/api-platform/demo/blob/main/api/tests/Api/RefreshDatabaseTrait.php)